drivers/spi/spi_mcux_lpspi.c

```diff
@@ -170,6 +170,12 @@ static int spi_mcux_configure(struct device *dev,
 		: kLPSPI_MsbFirst;
 
 	master_config.baudRate = spi_cfg->frequency;
+	master_config.betweenTransferDelayInNanoSec =
+		1000000000U / master_config.baudRate * 2;
```

Comment:
what is 1000000000 representing? 
If it's the number of nano seconds per seconds, please use NSEC_PER_SEC instead.

Reply:
Agreed

Reply:
Thanks for your review.
This code refers to the writing of fsl_lpspi.c LPSPI_MasterGetDefaultConfig, which has now been replaced by NSEC_PER_SEC